### DISCUSSION TITLE
account suspended

### DISCUSSION THREAD

**Discussion by karrthick1996** [community/community#110593] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

Hello,

on 03/01/2024, My github account has been suspended and I am reaching out to support via email but there is no response.
this message appears "Access to your account has been suspended due to a violation of our [Terms of Service].
Please [contact support] for more information."
The github account has everything which I have achieved in tech till now.

Deleted account username - Saravana28799
Ticket ID: 2643554

I can't lose the account. I request anyone to please suggest what should I do in this case?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @karrthick1996, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 
As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
